composer.json

```diff
@@ -18,11 +18,12 @@
         "aws/aws-sdk-php": "^3.209"
     },
     "require-dev" : {
-        "squizlabs/php_codesniffer" : "^3.5",
-        "phpunit/phpunit" : "9.4.4",
+        "squizlabs/php_codesniffer" : "^3.7",
+        "phpunit/phpunit" : "11.5.0",
+	"phpunit/php-code-coverage" : "^11.0",
         "phan/phan": "^5.0",
         "phpstan/phpstan": "^1.4",
-        "slevomat/coding-standard": "^6.4",
```

Comment:
If these changes are causing large amounts of changes, I think they should be done 1 at a time in different pull requests to make it easier to review and test them individually.